test/index.test.js

```diff
@@ -56,10 +56,25 @@ function clearDir (dir) {
 for (const integrationTest of fs.readdirSync(__dirname + "/integration")) {
   // ignore e.g.: `.json` files
   if (!/\.(mjs|tsx?|js)$/.test(integrationTest)) continue;
+
+  // TODO: Reenable this test once https://github.com/webpack/webpack/issues/8509 is fixed
+  if (integrationTest.endsWith('path-platform.js')) continue;
+
   it(`should evaluate ${integrationTest} without errors`, async () => {
+    if (global.gc) {
+      global.gc();
+      console.log(`GC Completed, Heap Size: ${process.memoryUsage().heapUsed / 1024 ** 2} MB`);
+    }
+
     const { code, map, assets } = await ncc(
       __dirname + "/integration/" + integrationTest,
-      { minify: true, sourceMap: true }
+      {
+        // enabling cache stops saslprep.js test from working
+        // which seems like a Webpack 5 cache bug...
+        cache: false,
```

Comment:
The `saslprep.js` test was failing until I disabled the cache explicitly. The failure was that the asset wasn't being emitted.

When the saslprep.js test is run on its own in isolation it passes fine, regardless of cache state, so it seems like some kind of interaction between webpack cache and previous build memoization.

Because it works in isolation fine, and this is the only test that fails, I don't think this is as bad as it sounds, but it's worth bearing in mind.